workers/unity/Assets/Gdk/Core/Config/ReceptionistConfig.cs

```diff
@@ -23,5 +23,14 @@ public static ReceptionistConfig CreateConnectionConfigFromCommandLine(Dictionar
                 parsedArgs, RuntimeConfigNames.LinkProtocol, RuntimeConfigDefaults.LinkProtocol);
             return config;
         }
+
+        public static ReceptionistConfig CreateConnectionConfigForAndroidEmulator()
+        {
+            return new ReceptionistConfig
+            {
+                ReceptionistHost = "10.0.2.2",
```

Comment:
Make this configurable?

Reply:
It is a default alias for the IP of the host device the emulator is running on. By configurable do you mean moving it to a global variable at the top of the file or something else?

Reply:
Can make it a global for now with an informative name, like HostDeviceIP or something?